cmd/amp/platform_sendVerificationEmail.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"github.com/appcelerator/amp/api/client"
+	"github.com/appcelerator/amp/pkg/ampmail"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// PlatformSendVerificationEmail to send a verification email to a user
+var PlatformSendVerificationEmail = &cobra.Command{
```

Comment:
Maybe it would be better to move these commands on a specific branch, separate from the PR